sdgym/synthesizers/independent.py

```diff
@@ -2,32 +2,34 @@
 from sdv.metadata import Table
 from sklearn.mixture import GaussianMixture
 
-from sdgym.synthesizers.base import MultiSingleTableBaseline
+from sdgym.synthesizers.base import MultiSingleTableBaselineSynthesizer
 
 
-class Independent(MultiSingleTableBaseline):
+class IndependentSynthesizer(MultiSingleTableBaselineSynthesizer):
     """Synthesizer that learns each column independently.
 
     Categorical columns are sampled using empirical frequencies.
     Continuous columns are learned and sampled using a GMM.
     """
 
-    @staticmethod
-    def _fit_sample(real_data, metadata):
+    def _get_trained_synthesizer(self, real_data, metadata):
```

Comment:
why is this called `get_trained_synthesizer` if it returns metadata and transformed data?

Reply:
This is kind of a special case synthesizer, because we created one to sample columns independently, and there is no actual model that it's referencing. Instead of making a model class that stores the metadata, etc, I just returned it as a tuple.